docs/platform/linode-images.md

```diff
@@ -13,7 +13,10 @@ published: 'Thursday, September 25th, 2014'
 title: Linode Images
 ---
 
-Linode Images allows you to take snapshots of your disks, and then deploy them to any Linode under your account. This can be useful for bootstrapping a master image for a large deployment, or retaining a disk for a configuration that you may not need running, but wish to return to in the future. Linode Images will be retained whether or not you have an active Linode on your account, which also makes them useful for long term storage of a template that you may need in the future.
+Linode Images allows you to take snapshots of your disks, and then deploy them to any Linode under your account. This can be useful for bootstrapping a master image for a large deployment, or retaining a disk for a configuration that you may not need running, but wish to return to in the future. Linode Images will be retained whether or not you have an active Linode on your account, which also makes them useful for long term storage of a private template that you may need in the future. There is no additional charge to store images for Linode users, with a limit of 10 images per account.
+
+ {: .note }
+>If you delete an Linode, the IP address will be removed. So if you redeployed an image, you'd get a new IP address.
```

Comment:
While that statement is technically true, it's completely outside of the realm of an image. An IP address would never be in an image on any platform.

The IP can change, the MAC can change, FQDN can change, resources, etc. Much more than simply an IP address.